### PR TITLE
Display: Add ability to skip sections

### PR DESCRIPTION
**Pull request by andrewjj20**
The KVM and MSR sections can now be skipped when display is run.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -36,8 +36,13 @@ enum CommandOpts {
 
 #[derive(StructOpt)]
```
Comment (cmouzaoui) [inline src/main.rs:37]: it appears that `structopt` is being deprecated in favor of `clap`: https://docs.rs/structopt/latest/structopt/

File: src/main.rs [hunk 2]
```diff
@@ -72,7 +77,7 @@ impl Command for Disp {
                 }
             }
 
-            if MSRDesc::is_availible() {
+            if MSRDesc::is_availible() && !self.skip_msr {
```
Comment (cmouzaoui) [inline src/main.rs:80]: availible -> available

**Review by cmouzaoui — CHANGES_REQUESTED** [review]

File: src/main.rs (hunk 1 above)
Comment (andrewjj20) [reply src/main.rs:37]: see https://github.com/ScaleComputing/cpuinfo/issues/12

File: src/main.rs (hunk 2 above)
Comment (andrewjj20) [reply src/main.rs]: should be fixed now.

**Review by cmouzaoui — APPROVED** [review]

**Pull request merged**